src/css/footer.css

```diff
@@ -1,109 +1,110 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding: 0 30px;
+    padding: 0 1.875rem;
     justify-content: flex-start;
 }
-.footer__heading {
+
+.footer-heading {
     font-size: 1.5rem;
     font-weight: 700;
-    margin-bottom: 20px;
+    margin-bottom: 1.25rem;
     color: #242323;
     font-family: 'Playfair Display', serif;
 }
-.footer__nav-list {
+
+.footer-nav-list {
     list-style: none;
     padding: 0;
     margin: 0;
     display: flex;
     flex-direction: column;
-    gap: 10px;
+    gap: 0.625rem;
 }
-.footer__nav-list li a {
+
+.footer-nav-list li a {
     text-decoration: none;
     color: #24232396;
-    font-size: 18px;
+    font-size: 1.125rem;
     font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
     font-weight: 500;
     transition: color 0.2s;
 }
-.footer__nav-list li a:hover {
+
+.footer-nav-list li a:hover {
     color: #000;
 }
 
 /* Footer grid layout */
 section#links-footer {
-    padding: 100px 100px 50px 100px;
+    padding: 6.25rem 6.25rem 3.125rem 6.25rem;
     background-color: #F6F6F6;
 }
 
-    section#links-footer .section-container {
-        display: grid;
-        grid-template-columns: 1fr 1fr 1fr 1fr;
-        grid-template-rows: 1fr;
-        align-items: start;
-        background-color: #F6F6F6;
-    }
+section#links-footer .section-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr;
+    align-items: start;
+    background-color: #F6F6F6;
+}
 
-    /* BEM: wrapper voor navigatie in footer */
-    /* .footer__nav-wrapper {
-        
-    } */
-    .footer__heading {
-        font-size: 25px;
-        font-weight: 700;
-        font-family: 'Playfair Display';
-        color: black;
-        margin-bottom: 20px;
-    }
-    .footer__nav-list,
-    .footer__quick-list,
-    .footer__open-list,
-    .footer__consult-list {
-        list-style: none;
-        margin: 0;
-        padding: 0;
-    }
-    .footer__nav-list li,
-    .footer__quick-list li,
-    .footer__open-list li,
-    .footer__consult-list li {
-        margin-bottom: 15px;
-    }
+.footer-heading {
+    font-size: 1.5625rem;
+    font-weight: 700;
+    font-family: 'Playfair Display';
+    color: black;
+    margin-bottom: 1.25rem;
+}
+
+.footer-nav-list,
+.footer-quick-list,
+.footer-open-list,
+.footer-consult-list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+
+.footer-nav-list li,
+.footer-quick-list li,
+.footer-open-list li,
+.footer-consult-list li {
+    margin-bottom: 0.9375rem;
+}
 
 .footer-quick-links .inner a:hover,
 .footer-nav-links .inner a:hover,
 .footer-logo-container a:hover,
-.operational .inner a.link-hover:hover{
+.operational .inner a.link-hover:hover {
     color: black;
 }
 
-
-#links-footer .section-container > div:not(:first-child) {
-    border-left: 2px solid #0000003b;
-    padding-left: 30px;
-    padding-right: 30px;
+section#links-footer .section-container>div:not(:first-child) {
```

Comment:
Niet stylen op een #ID selector. Deze selector vind ik verder ook aan de lange kant. 

Als je de `div` vervangt voor een `.class` dan kan je selector dit ongeveer zijn: `.footer-section:not(:first-child) {`